rpxdock/filter/filter_body.py

```diff
@@ -0,0 +1,36 @@
+from rpxdock.app import dock
+import rpxdock.homog as hm
+import logging
+
+def filter_body(body, xforms, arch):
+   logging.debug(f"\nArchitecture for filtering {arch}\n")
+   arch = arch.upper()
+   if arch.startswith('C') and len(arch) == 2:
+      #Do cx stuff
+      xforms = xforms.reshape(-1, 4, 4)
+      sym = "C%i" % i if isinstance(arch, int) else arch
+      symrot = hm.hrot([0, 0, 1], 360 / int(sym[1:]), degrees=True)
+      xsym = symrot @ xforms
+      pos1 = xforms
+      pos2 = xsym
+      body1 = body
+      body2 = body
+      
```

Comment:
Let's do this in the next version. It's going to take a bit more work than I thought and I'd really like to close this branch. Cleaning this up is important but doesn't change the functionality, it can be its own project.